docs/lesson/improvements.rst

```diff
@@ -0,0 +1,290 @@
+-----------------------
+
+Learners also have the option to suggest an *edit* on one or more cards, e.g., a change in the content, formatting etc. This comes through as a *Suggestion* task in the Improvements screen.
+
+.. figure:: /images/suggestion_task.png
+   :alt: Suggestion task
+   :scale: 40 %
+
+   *Fig. 3*
+
+1. Click the |reviewthread| button and then the |reviewsuggestion| button to action this suggestion.
+
+.. |reviewsuggestion| image:: /images/review_suggestion_button.png
+                      :alt: Review Suggestion button
+                      :scale: 35 %
+
+2. After reviewing the changes suggested by the learner, you can either accept or reject the suggestion.
+
+.. figure:: /images/review_suggestion.png
+   :alt: Review suggestion box
+   :scale: 40 % 
+
+   *Fig. 4*
+
+3. Enter a new message if you wish to add to the message thread and change the status of the suggestion as required. Finally, click the |sendandclose| button. 
+
+.. |sendandclose| image:: /images/send_and_close.png
+                  :alt: Send and Close button
+                  :scale: 40 %
+
+.. figure:: /images/suggestion_message.png
+   :alt: Suggestion message box
+   :scale: 30 %
+
+   *Fig. 5*
+
+.. _answerdetails:
+
+Answer details for a card
+--------------------------
+
+If you checkmarked the **Solicit Answer Details** box while creating your exploration, Oppia will occasionally ask a learner to explain why they chose a certain answer for that card.
+
+.. figure:: /images/solicit_answer_details.png
+   :alt: Solicit answer details checkbox
+   :scale: 40 %
+
+   *Fig. 6*
+
+Whenever a learner explains why they chose their answer for a card, you will see the details in the Improvements screen as shown below.
+
+.. figure:: /images/answer_details_task.png
+   :alt: Answer details task
+   :scale: 40 %
+
+   *Fig. 7*
+
+1. To review the details, click the |reviewanswerdetails| button and click under **Answer details** to expand the comment.
+
+.. |reviewanswerdetails| image:: /images/review_answer_details.png
+                         :alt: Review Answer Details box
+                         :scale: 40 %
+
+.. figure:: /images/answer_details_review.png
+   :alt: Review answer details
+   :scale: 40 %
+
+   *Fig. 8*
+
+
+2. To delete the **Answer details** task, checkmark the **Select** box and click on the |deleteitems| box.
+
+.. |deleteitems| image:: /images/delete_selected_items.png
+                 :alt: Delete Selected items box
+                 :scale: 40 %
+
+
+.. admonition:: An example on improving an exploration
+
+   Let's say you asked the question: **Which is greater: 0.834 or 0.2?** and requested an explanation from the learner for their answer. The learner selected the right answer but explained they chose it because it was a longer number. This is obviously a misconception, so you may then decide to include additional cards beforehand or :ref:`branch out <addbranch>` to focus on place value or the decimal-fraction link.
```

Comment:
Please add comma: "The learner selected the right answer, but explained they..."

Reply:
Done